integ/locking_test.go

```diff
@@ -116,7 +116,7 @@ func doTestLockingWorkflow(t *testing.T, backendType service.BackendType, config
 					ValueType: iwfidl.INT.Ptr(),
 				},
 			}
-			time.Sleep(time.Second * 1)
+			time.Sleep(time.Second * 2)
```

Comment:
Test passes locally with no change. I had to bump it to make the test pass in Github CI

Reply:
I am still not sure why we need more sleep. I remember there is a new behavior in temporal cloud that they now enforce a backoff of 1s for each continueAsNew to prevent having too fast continueAsNew which cause hot partition in server. Maybe that's the cause. we can dig into the release logs to confirm, or maybe ask them

Reply:
actually the minimum backoff has been there for a long time: https://github.com/temporalio/temporal/blame/fad6bdc0e9c0911f28829f3c47285357554e2567/service/history/workflow/mutable_state_impl.go#L1703

Reply:
I'm looking into this, but it's not easy to troubleshoot since it passes locally

Reply:
I just looked into that, looks like there is a test panic at timeout for the cadence cases